lib/libutils/ext/include/util.h

```diff
@@ -188,7 +188,34 @@ static inline void reg_pair_from_64(uint64_t val, uint32_t *reg0,
 	*reg1 = low32_from_64(val);
 }
 
-/* Get and set bit fields  */
+/**
+ * @brief Functions to get and set bit fields in a 32/64-bit register.
+ *
+ * These inline functions allow setting and extracting specific bits in
+ * a register (`reg`) according to a given mask (`mask`). The mask
+ * specifies which bits in the register should be updated or extracted.
+ *
+ * - `set_field`: Modifies specific bits in a register by clearing the
+ *   bits specified by the mask and then setting these bits to the new
+ *   value (`val`).
+ * - `get_field`: Extracts the value of specific bits in a register by
+ *   isolating the bits specified by the mask and then shifting them to
+ *   the rightmost position.
+ *
+ * @param reg The original 32-bit or 64-bit register value.
```

Comment:
s/@​param /@/
If the description doesn't fit on a single line, please indent the following lines to align with the first word of the description.

Reply:
Hello Jens,

Thank you for your remarks. I have taken them into account and I've improved the rest of the text to make it coherent with your recommendations.

cheers